content/docs/howto/negate-a-disjunction/en.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Negate a disjunction
+tags:
+- validation
+- language
+- commented cue
+authors:
+- hansbogert
+relatedIssues:
+- https://github.com/cue-lang/cue/discussions/3229
+- https://github.com/cue-lang/docs-and-content/issues/160
+toc_hide: true
+---
+
+This [Commented CUE]({{< relref "docs/howto#commented-cue-guides" >}})
+demonstrates how to negate a disjunction.
+
+{{{with code "en" "example"}}}
+! exec cue vet example.cue
```

Comment:
When we have only a single CUE package in scope (in a directory) we omit the CUE file (and package name) that's being evaluated, from the command invocation.